src/goalServices/getGoalIdsBySimiliarity.test.js

```diff
@@ -1,8 +1,10 @@
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
Not sure if there is anyway around this like adding an additional test for the if statment.